docs/pages/admin-guides/access-controls/access-requests/automatic-reviews.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: Configure Automatic Reviews
+description: Describes how to configure Access Monitoring Rules for Automatic Reviews.
+---
+
+Teleport supports automatic reviews of Access Requests for Teleport roles. This
+feature enables teams to enforce a zero standing privilege policy, while
+still allowing users to receive temporary access without manual approval.
+
+## How it works
+
+Automatic reviews are triggered by Access Monitoring Rules. These rules instruct
+Teleport to monitor specific Access Requests and automatically submit a review
+when certain conditions are met. The conditions can be configured based on the
+requested roles, and based on the requesting user's Teleport traits.
+
+For example, an Access Monitoring Rule can configure automatic Access Request
+approval when a user with the Teleport traits `team: demo` requests access to
+the `access` role.
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+## Step 1/4. Create a requester role and user
+
+In this example, we'll first create:
+- A role named `demo-access-request`, which allows requesting access to the
+`access` role.
+- A user named `demo-access-requester`, assigned the above role.
+
+### Create the role
+
+Create a role configuration file named `demo-role.yaml`:
+```yaml
+# demo-role.yaml
+kind: role
+version: v7
+metadata:
+  name: demo-access-request
+spec:
+  allow:
+    request:
+      roles:
+      - access
+      search_as_roles:
+      - access
+```
+
+Create the role with:
+```sh
+$ tctl create demo-role.yaml
+```
+
+### Create the user
+
+Use the following command to create the user and assign the role:
+```sh
+$ tctl users add --roles=demo-access-request demo-access-requester
+```
+
+Alternatively, you can assign the role after creating the user:
+
+  (!docs/pages/includes/add-role-to-user.mdx role="demo-access-request" user="\`demo-access-requester\`"!)
+
+## Step 2/4. Assign user traits
+
+To allow automatic review rules to evaluate the requesting user, assign them
+traits via the Teleport Web UI.
+
+1. Go to **Zero Trust Access** -> **Users**
+2. Next to `demo-access-requester`, click **Options** -> **Edit...**
+3. Click **Add user trait**, and set:
+   - Key: `team`
+   - Value: `demo`
+4. Click **Save**
+5. Verify that the user has been updated with the desired trait.
+
+![Edit User Traits](../../../../img/access-controls/automatic-reviews/users-edit.png)
```

Comment:
```suggestion
![Edit User Traits](../../../../img/access-controls/automatic-reviews/users-edit.png)

When adding user traits, you can enter any keys and values. The user trait form does not support wildcards or regular expressions.
```

Suggested because (as far as I recall) some Web UI forms do support these, so I thought we could make the constraints explicit.